internal/cache/file/downloader/parallel_downloads_job_test.go

```diff
@@ -47,7 +47,7 @@ func init() { RegisterTestSuite(&parallelDownloaderTest{}) }
 
 func (dt *parallelDownloaderTest) SetUp(*TestInfo) {
 	dt.defaultFileCacheConfig = &config.FileCacheConfig{EnableParallelDownloads: true,
-		DownloadParallelismPerFile: 3, ReadRequestSizeMB: 3, EnableCrcCheck: true}
+		ParallelDownloadsPerFile: 5, DownloadChunkSizeMB: 2, EnableCRC: true}
```

Comment:
Why do we need to change these values?

Reply:
We don't need to change these values, I have put them back.